Header header
int32 voxel_ix
int32 voxel_iy
int32 voxel_iz
float64 global_x
float64 global_y
float64 global_z
